### PR TITLE
1.0/269 email voucher history

### PR DESCRIPTION
**Pull request by katjam**
Hack by me to get the button working. Please tidy before or after merge.

### CODE REVIEW

**Review by gingerbreadpup — APPROVED** [review]
:shipit:

File: src/pages/Account.vue [hunk 1]
```diff
@@ -71,6 +72,18 @@
         created: function () {
 
         },
+        methods: {
+            onRequestVoucherHistoryEmail(submittedOn = null) {
+                NetMgr.apiPost('/traders/' + Store.trader.id + '/voucher-history-email?submission_date=' + submittedOn,
+                    function (response) {
+                        if (success) { success(response) }
```
Comment (charlesstrange2) [inline src/pages/Account.vue:79]: unneeded function doesn't take a callback

File: src/pages/Account.vue [hunk 2]
```diff
@@ -71,6 +72,18 @@
...
+                    },
+                    function () {
+                        if (failure) { failure(error) }
```
Comment (charlesstrange2) [inline src/pages/Account.vue:82]: unnneded, function doesnt take a callback

File: src/pages/Account.vue [hunk 3]
```diff
@@ -71,6 +72,18 @@
         created: function () {
 
         },
+        methods: {
+            onRequestVoucherHistoryEmail(submittedOn = null) {
```
Comment (charlesstrange2) [inline src/pages/Account.vue:76]: if it's an onclick handler, i'd expect submitteOn to be stuffed with an event?

**Review by charlesstrange2 — CHANGES_REQUESTED** [review]
Is that not receiving an onclick event in that submittedOn variable?

**Comment by katjam** [comment]
@charlesstrange2 or @gingerbreadpup 

If you can fix it great... All I know is that it currently works :) even if it is badly written.

The next step for me is to add the submitted variable to the little icons that ask for emails per submitted date -which is where that param will be used. I am using the null value as a default to mean 'all history' but the other buttons will submit a value.

**Pull request closed without merging**